config/jobs/kubernetes/sig-security/snyk-scan.yaml

```
@@ -0,0 +1,37 @@
+#  Periodic CI job for running snyk scans against k/k master
+# - It installs snyk CLI and requires 'snyk_token' secret available
+#   in test infra with key name 'SNYK_TOKEN'. This secret is used to
+#   populate env var 'SNYK_TOKEN', required for snyk CLI auth.
+# - Licenses and few false positive deps (eg version '0.0.0') are
+#   filtered from the snyk scan results and printed on stdout as well
+#   as "${ARTIFACTS}/snyk_results.json"
+periodics:
+  - interval: 6h
+    name: snyk-scan-kubernetes-master-periodical
```

Run in k8s-infra-prow-build trusted, and follow job naming convention of kubernetes periodics being named as `ci-kubernetes-{foo}-{branch}`
```suggestion
    cluster: k8s-infra-prow-build-trust
    name: ci-kubernetes-snyk-master
```

---

done